website/docs/api/reform-helpers.md

```diff
@@ -0,0 +1,23 @@
+---
+title: ReForm.Helpers
+id: reform-helpers
+slug: /reform-helpers
+---
+
+`ReForm.Helpers` basically provides some helpers to deal with form elements, fields, etc: 
```

Comment:
I wonder that we maybe we'll to move this to ReForm.Web.Helpers etc so people don't try to use them in RN or assume this a web only lib